### PR TITLE
Add types definitions for xk6-sql

### PR DESCRIPTION
**Pull request by leandrodotec**
Adding definitions for k6/x/sql plugin: https://github.com/grafana/xk6-sql

Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/xk6-sql-browser/tslint.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+	 "npm-naming": false
```
Comment (DangerBotOSS) [inline types/xk6-sql-browser/tslint.json:3]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

**Review by PugAxe — APPROVED** [review]
lgtm

**Comment by leandrodotec** [comment]
Hi @PugAxe, can you re-approve again? I forgot one little indentation problem.

**Review by PugAxe — APPROVED** [review]

File: types/xk6-sql-browser/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
+// Type definitions for non-npm package xk6-sql 0.1
+// Project: https://github.com/grafana/xk6-sql
+// Definitions by: Leandro Gomes <https://github.com/leandrodotec>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.8
```
Comment (peterblazejewicz) [inline types/xk6-sql-browser/index.d.ts:5]:
```suggestion
```
that version is no longer enforced or specifically tested here, just remove this line.

File: types/xk6-sql-browser/tslint.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "npm-naming": false
```
Comment (sheetalkamat) [inline types/xk6-sql-browser/tslint.json:3]:
```suggestion
```
This needs to be removed and if any error fixed.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/xk6-sql-browser/tslint.json (hunk 3 above)
Comment (leandrodotec) [reply types/xk6-sql-browser/tslint.json]: Done.

**Review by sheetalkamat — APPROVED** [review]

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@leandrodotec thanks!

**Comment by leandrodotec** [comment]
Ready to merge

**Pull request merged**